docker/pypi/wmagent-mariadb/Dockerfile

```diff
@@ -63,9 +59,6 @@ alias manage=$MDB_MANAGE_DIR/manage
 export PS1="(MariaDB-$MDB_TAG) [\u@\h:\W]\$([[ \$(id -u) -eq 0 ]] && echo \# || echo \$) "
 EOF
```

Comment:
look at my comment referring the respective lines at the CouchDB container.

Reply:
Todor, I resolved previous comments, but I'm not sure about this one, please clarify it since I didn't touch mariadb prompt, and only added CouchDB one.